chartpress.py

```diff
@@ -295,9 +299,12 @@ def build_values(name, values_mods):
                 )
 
             mod_obj['tag'] = value['tag']
+        elif isinstance(mod_obj, str) and set(value.keys()) == {'repository', 'tag'}:
+            # scalar image string, not dict with separate repository, tag keys
+            parent[parts[-1]] = "{repository}:{tag}".format(**value)
```

Comment:
This may fail if we want to update a numbered item in an array I think. We could use the variable `p` instead if we want to avoid that, or repeat this logic.

```python
            if p.isdigit():
                # integers are indices in lists
                p = int(p)
```

I don't see that use case at the moment though, but perhaps.

Reply:
This was my quick check to figure out if you could reference a out-of-for-loop-scope variable.

![image](https://user-images.githubusercontent.com/3837114/67166353-af17f800-f38e-11e9-934c-b398b637030c.png)

Reply:
Good catch! Fixed.